rails/config/application.rb

```diff
@@ -43,7 +46,7 @@ class Application < Rails::Application
 
     config.autoload_paths += Dir["#{config.root}/lib/**/"] # include lib and all subdirectories
     config.autoload_paths += Dir["#{config.root}/app/pdfs/**/"] # include app/reports and all subdirectories
```

Comment:
I didn't even know we had a pdfs directory!  I find it incredible that it has to be included in the autoload_paths.

Realize that is *not* part of this PR, but just making a note of it here.  This must be a mistake?  We should investigate later.